components/product/ProductCard/components/ImageAndTag.tsx

```diff
@@ -0,0 +1,59 @@
+import ImageDecoded from "deco-sites/persono/components/ui/ImageDecoded.tsx";
+
+interface Props {
+  hasNews: boolean;
+  hasDiscount: boolean;
+  preload?: boolean;
+  discount: number;
+  imageAlt?: string;
+  imageBaseUrl?: string;
+  newsTitle: string;
+  imageSrc: string;
+}
+
+export const ImageAndTag = ({
+  hasDiscount,
+  hasNews,
+  discount,
+  preload,
+  imageAlt = "",
+  imageBaseUrl,
+  imageSrc,
+  newsTitle,
+}: Props) => {
+  const WIDTH = 287;
+  const HEIGHT = 287;
+
+  const showHighlightTag = hasNews || hasDiscount;
+
+  return (
+    <figure class="relative " style={{ aspectRatio: `${WIDTH} / ${HEIGHT}` }}>
+      <div class="w-full">
+        {showHighlightTag && (
+          <span
+            class={`py-1 px-3 flex absolute  rounded-br-xl justify-center items-center text-sm ${
+              hasNews ? "bg-blueNew" : "bg-black"
```

Comment:
As cores semanticas ja foram cadastradas no Theme: https://admin.deco.cx/sites/persono/blocks/Default%20Theme
```suggestion
              hasNews ? "bg-primary" : "bg-secondary"
```

Reply:
Mais importante que isso, as cores das Tags vao vir nos produtos. Basicamente voce vai receber um additionalProperty que é uma Tag. Fala com o Sacci sobre isso, ele ja deve ter isso pronto para voce deixar a logica pronta. Qualquer coisa fale comigo